src/SUMMARY.md

```diff
@@ -5,6 +5,7 @@
   - [Gaining Access](concepts/gaining_access.md)
   - [Analysis Quick Start](concepts/analysis_intro.md)
   - [Tools for Data Analysis](tools/interfaces.md)
+    - [GLAM](tools/glam.md)
```

Comment:
We're trying to keep the introduction section pretty clean (I made a pretty major overhaul last summer), so I'd prefer to keep documentation on specific tools either in the cookbook or reference section. If you haven't seen it yet, the style guide has more details about this: https://docs.telemetry.mozilla.org/contributing/style_guide.html

It looks like most of this could be considered a "cookbook". Could you move it to "Tutorials & Cookbooks"? I'd put it right after "Accessing Public Data". IMO GLAM should be one of the first things we want to encourage people with a data-related question to try using.